samples/entity-framework-core/SqlPersistence_8/Endpoint.SqlPersistence/Handlers/CompleteOrderHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using NServiceBus;
+using NServiceBus.Logging;
+
+public class CompleteOrderHandler(ReceiverDataContext dataContext)
+        : IHandleMessages<CompleteOrder>
+{
+    static readonly ILog log = LogManager.GetLogger<CompleteOrderHandler>();
+
+    public async Task Handle(CompleteOrder message, IMessageHandlerContext context)
+    {
+        var order = await dataContext.Orders.FindAsync(message.OrderId, context.CancellationToken)
```

Comment:
Nit: should we skip .ConfigureAwait(false) for new samples we add?

Reply:
I haven't been removing it on prior samples.

Reply:
We should consider removing `ConfigureAwait`s but should do that in a separate pass, possibly with an integrity test to enforce it